kyber/__init__.py

```diff
@@ -44,6 +45,8 @@ def deploy_app(tag, force):
         click.echo("Can't find a docker for {}\naborting..".format(app.image))
         return
 
+    click.confirm("About to deploy {} to {}, continue?".format(tag, context.name), abort=True)
```

Comment:
This prompt should probably be before the ECR lookup because it's slow.  People will do "kb deploy" then switch to something else, and then see the prompt 5 minutes later.  Also should we display the kube api context data?  Also we should have an option to disable this prompt if we want to automate deployments using kb.

A really cool stretch would be to do an updating output line below the prompt, indicating whether the chosen tag is deployable, and abort immediately if it isn't:
```
$ kb deploy
About to deploy git_23123234 to takumi-server, continue [y/N] ?
[...checking deployability...]
...
[OK]
```

or:
```
$ kb deploy
About to deploy git_23123234 to takumi-server, continue [y/N] ?
[...checking deployability...]
...
[NOT OK]
aborting
```

etc..